data/data/google/network/master-lb.tf

```diff
@@ -0,0 +1,62 @@
+
+resource "google_compute_global_address" "master" {
+  name = "${var.cluster_name}-master"
+}
+
+resource "google_compute_health_check" "masters" {
+  name = "${var.cluster_name}-masters"
+
+  https_health_check {
+    port = 6443
+    request_path = "/healthz"
+  }
+}
+
+resource "google_compute_backend_service" "masters" {
+  name        = "${var.cluster_name}-masters"
+  port_name   = "https"
+  protocol    = "TCP"
+  timeout_sec = 120
+
+  backend {
+    group = "${var.bootstrap_instance_group}"
+  }
+  // gross, in 0.12 this hopefully gets better
+  backend {
+    group = "${var.master_instance_groups[0]}"
+  }
+  backend {
+    group = "${var.master_instance_groups[1]}"
+  }
+  backend {
+    group = "${var.master_instance_groups[2]}"
+  }
+
+  health_checks = ["${google_compute_health_check.masters.self_link}"]
+}
+
+resource "google_compute_target_tcp_proxy" "master" {
```

Comment:
It looks like this load balancer is Internet facing. Do you have plans to support a GCP ILB at some point in the future?

https://cloud.google.com/load-balancing/docs/internal/